src/android/toga_android/app.py

```diff
@@ -1,9 +1,14 @@
 import asyncio
+import toga
 
 from rubicon.java import android_events
 from toga.handlers import wrapped_handler
 
 from .libs.activity import IPythonApp, MainActivity
+from .libs.android.content import Intent
+from .libs.android.net import Uri
```

Comment:
Are we missing a file here? This PR doesn't define `android.net` AFAICT....

Reply:
Sorry about this! I *did* test everything and it *did* work. 

I first based the branch on a wrong parent branch and then had to redo all the changes on the new branch commands-for-android. Seems that I missed the `net` module then, but on my file system, the file was still there when I tested it all (again)...

But now, it seems that I don't need it, because no Android specific implementation is needed.

Reply:
OK, reverted `visit_homepage` back to the original version of toga/app.py:
```
    def visit_homepage(self):
        if self.home_page is not None:
                webbrowser.open(self.home_page)
```
But now, we have the problem, that nothing happens on Android. No webbrowser opens, no error message. In logcat, I only see this:
```
D/MainActivity: onOptionsItemSelected() start
D/Python: Native invocation 3542967056 :: onOptionsItemSelected
D/Python: Native invocation done.
D/MainActivity: onOptionsItemSelected() complete
```